### PR TITLE
add 119 pascals-triangle-ii

### LINKED ISSUE #55: add 119 Pascal's Triangle II
## Summary
add 119

## Goal
add 119

## Todo
- [ ] add 119
- [ ] write time and space complexity

## Deadline
5/2

## Parent issue
None

## References
None

## Notes
None

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 5m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 119.pascals-triangle-ii.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        time: O(rowIndex^2)
+        space: O(rowIndex^2)
+        """
+        cur_Row_list = [1]
+        for Row in range(2, rowIndex + 2):
+            cur_Row_list = [1]
+            for cul_num in range(1, Row - 1):
+                cur_Row_list.append(pre_Row_list[cul_num - 1] + pre_Row_list[cul_num])
+            cur_Row_list.append(1)
+            pre_Row_list = cur_Row_list
```
Comment (ryuji0123) [inline 119.pascals-triangle-ii.py:20]: Insert a line to show divided logical sections.

**Review by ryuji0123 — COMMENTED** [review]
See "add 118"

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
Check the last comment.

**Pull request closed without merging**